ui/app/templates/vault/cluster/settings/auth/configure.hbs

```diff
@@ -12,22 +12,19 @@
       Configure {{get (find-by "type" model.type (mountable-auth-methods)) "displayName"}}
     </h1>
   </p.levelLeft>
-  <p.levelRight>
-    <div class="field is-grouped">
-      <div class="control">
-        {{#link-to
-          "vault.cluster.access.method"
-          model.id
-          class="button is-ghost has-icon-right is-compact"
-          data-test-backend-view-link=true
-        }}
-          View method
-          {{i-con glyph="chevron-right" size=11}}
-        {{/link-to}}
-      </div>
-    </div>
-  </p.levelRight>
 </PageHeader>
 
 {{section-tabs model}}
+
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink
+      @params={{array "vault.cluster.access.method" model.id}}
+      @data-test-backend-view-link=true
```

Comment:
data attributes don't need the `@` those and `class` and others (`aria-*`) attributes get treated as normal HTML attributes with angle bracket syntax, they just get passed through and bound automatically - that's something you'd have to set up in curly bracket syntax

Reply:
Removing the `@` from these instances actually does stop it from feeding through, but that may be because of how we're passing them along in the template. I was able to remove them in the `ToolbarLink` component, but something about the `ToolbarSecretLink` component still needs them there to be able to pass them along to `SecretLink`. Oh well, at least we cleaned it up where we could.

Reply:
Oh I bet that has to do with this being tag-less - no tag so it probably skips the binding making these just regular component attrs. Sorry for the bad info!